signing.go

```diff
@@ -122,6 +124,7 @@ type asymmSigner struct {
 	prefix       string
 	created      int64
 	expires      int64
+	sshSigner    ssh.Signer
```

Comment:
Is this used? Or is only the sshSigner in `ed25519Algorithm` used? If this is not used, let's not add it here.

Reply:
No, you're correct, seems to be from some test I forgot to remove.